src/main_app/data/__init__.py

```
@@ -0,0 +1,25 @@
+
+logger = logging.getLogger(__name__)
+
+json_files = Path(__file__).parent.glob('*.json')
+
+data_list = {}
+
+for file in json_files:
+    try:
+        with open(file, 'r', encoding='utf-8') as f:
+            data_list[file.stem] = json.load(f)
+    except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)